tools/flang2/flang2exe/cgmain.cpp

```diff
@@ -2643,7 +2644,8 @@ get_omnipotent_pointer(LL_Module *module)
     const char *baseName = "Flang FAA";
     const char *const omniName = "unlimited ptr";
     const char *const unObjName = "unref ptr";
-    snprintf(baseBuff, 32, "%s %x", baseName, funcId);
+    snprintf(baseBuff, 32, "%s %zx %x", baseName,
```

Comment:
Are you sure it still fits into 32 characters?

Reply:
@pawosm-arm Check again?